openstates/scrape/event.py

```diff
@@ -3,17 +3,29 @@
 from ..utils import _make_pseudo_id
 from .base import BaseModel, SourceMixin, AssociatedLinkMixin, LinkMixin
 from .schemas.event import schema
+import uuid
 
 
 def calculate_window(*, base_day=None, days_before=30, days_after=90):
-    """ given details on a window, returns start & end dates for windowing purposes """
+    """given details on a window, returns start & end dates for windowing purposes"""
     if not base_day:
         base_day = date.today()
     start = base_day - timedelta(days=days_before)
     end = base_day + timedelta(days=days_after)
     return start, end
 
 
+def is_valid_uuid(val):
```

Comment:
Does it make sense to put this function in `scrape/event.py`? Or does it make more sense in a utility file somewhere that may be easier to discover for other uses?

Reply:
Makes sense to move it to utils